src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do list</title>
+  </head>
+  <body>
+    <section class="toDoSection">
+      <div class="toDoContainer">
+        <div class="toDoHeader">
+          <h1>Today's To Do</h1>
+          <i class="fa fa-refresh" aria-hidden="true"></i>
+        </div>
+        <form class="toDoForm" id="task-form">
+          <div class="inputArea">
+            <input id="title" type="text" placeholder="Add to list..." />
+            <i class="fa fa-sign-in enter-icon" aria-hidden="true"></i>
+          </div>
+        </form>
+        <ul class="toDoItem"></ul>
+        <button class="clearTask" type="button">
+          Clear all
+        </button>
+      </div>
+    </section>
+    <!-- <script src="main.js"></script> -->
```

Comment:
[OPTIONAL] Please remove unwanted commented code from your codebase because it makes your codebase messy if you want to check your previous code then check GitHub commit history